src/main/scala/dpla/ingestion3/enrichments/WikiEntityEnrichment.scala

```diff
@@ -0,0 +1,104 @@
+package dpla.ingestion3.enrichments
+
+import dpla.ingestion3.model.{EdmAgent, URI}
+import dpla.ingestion3.utils.FileLoader
+
+/**
+  * Wikimedia entity enrichments
+  */
+class WikiEntityEnrichment extends FileLoader with VocabEnrichment[EdmAgent] {
+
+  protected val wikiUriBase = "https://wikidata.org/wiki/"
+
+  // Files to source vocabulary from
```

Comment:
I wonder if, in the long run, we should be thinking about the lifecycle of this data. How these records get created in Wikidata, and how we put them in the classpath before an ingest. Just a comment to think about, not really about this PR.

Reply:
I agree -- this did not seem ideal but good enough for this moment.